gruntfile.js

```diff
@@ -204,6 +209,33 @@ function shell_download_docker_binary(p, a) {
   }
 }
 
+function shell_download_docker_compose_binary(p, a) {
+  var ps = { windows: 'win', darwin: 'mac' };
+  var as = { amd64: 'x86_64', arm: 'armhf', arm64: 'aarch64' };
+  var ip = ps[p] === undefined ? p : ps[p];
+  var ia = as[a] === undefined ? a : as[a];
+  var binaryVersion = '<%= binaries.dockerComposeVersion %>';
+
+  if (p === 'linux' || p === 'mac') {
+    return [
+      'if [ -f dist/docker ]; then',
```

Comment:
maybe use template strings (backticks) instead of arrays? I think it's more readable... I wish we stop using grunt and just move to specific scripts...

Reply:
the logic likely meant to look for `dist/docker-compose` rather than `dist/docker`